index.html

```diff
@@ -79,126 +64,21 @@ <h1>I'm Anicet<br />Glad to see <br />you!</h1>
-                        <li id="javaScript"><a href="#">javaScript</a></li>
-                    </ul>
-                    <div class="to-project"><a href="#">See Project</a></div>
-                </div>
-            </div>
 
-
-            <div class="container container3">
-                <div class="Card1"><img src="./images/img1.png"
-                        alt="Here is the Facebook product designer Tenner Christemen" class="card1"></div>
-
-                <div class="p-desktop">
-                    <div>
-                        <h2>Facebook 360</h2>
-                    </div>
-                    <div class="lister">
-                        <p><strong>CANOPY</strong></p>
-                        <div class="point"></div>
-                        <P>Back End Dev</P>
-                        <div class="point"></div>
-                        <p>2015</p>
-                    </div>
-
-                    <p id="para1">Exploring the future of media in Facebook's first Virtual Reality app; a place to
-                        discover and enjoy 360 photos and videos on Gear VR.
-                    </p>
-                    <ul>
-                        <li id="html"><a href="#"> html</a></li>
-                        <li id="css"><a href="#">css</a></li>
-                        <li id="javaScript"><a href="#">javaScript</a></li>
-                    </ul>
-                    <div class="to-project"><a href="#">See Project</a></div>
-                </div>
-            </div>
-            <div class="container container4">
-                <div class="Card1"><img src="images/im2.png" alt="Here is our assistant for uber Navigation facilities"
-                        class="card1"></div>
-
-                <div class="p-desktop">
-                    <div>
-                        <h2>Uber Navigation</h2>
-                    </div>
-                    <div class="lister">
-                        <p><strong>CANOPY</strong></p>
-                        <div class="point"></div>
-                        <P>Back End Dev</P>
-                        <div class="point"></div>
-                        <p>2015</p>
-                    </div>
-
-                    <p id="para1">A smart assistant to make driving more safe, efficient, and fun by unlocking your most
-                        expensive computer: your car.
-                    </p>
-                    <ul>
-                        <li id="html"><a href="#"> html</a></li>
-                        <li id="css"><a href="#">css</a></li>
-                        <li id="javaScript"><a href="#">javaScript</a></li>
-                    </ul>
-                    <div class="to-project"><a href="#">See Project</a></div>
-                </div>
-            </div>
+        <div class="work-section" id="ancre1">
+        </div>
+        <div class="popup-container display-none">
+            <img id="closeBtn" src="images/exitIcon.png" alt="This button will close the pop-up window">
```

Comment:
- [ ] As already suggested by the previous reviewer, kindly make sure the close button for the modals is given a `position: fixed` property. You can remove the previous button image and put the event listener in the `x` instead.
- [ ] Also as suggested, kindly make sure the modal paragraph follows the correct color position, font size, font-weight, and font family as per the Figma design
- [ ] Also the project topic(in this case `CANOPY`) seems to be flowing out of the modal. Please fixt that also

![image](https://user-images.githubusercontent.com/43995830/157917218-c6f5a229-821a-4ddc-be2c-2b9ea232d85b.png)